Unable to build Blazor Client project: Input too long

Hello,

Since my last update of my repository, i'm unable to compile my project.
After some investigation, i see that there is a little more verbosity when compiling/stripping DLL for Blazor, and it seem that the shell limitation truncates the build parameters.

According to [this article](https://support.microsoft.com/en-us/help/830473/command-prompt-cmd-exe-command-line-string-limitation), the cmd args limitation is 8191 characters.

But the dotnet command issued by Blazor at build is issuing 8709 characters on my project, and so the build fail because arguments are truncated by the shell.

Here a sample of the failing command:

`dotnet "C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor.build\0.3.0-preview1-t000\targets\../tools/Microsoft.AspNetCore.Blazor.Build.dll" build obj\Debug\netstandard2.0\Allogaia.Web.Client.dll --html-page "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Web.Client\wwwroot/Index.html" --reference "Allogaia.AppResource.dll" --reference "Allogaia.AppResource.resources.dll" --reference "Allogaia.Common.dll" --reference "Daddoon.Blazor.dll" --reference "log4net.dll" --reference "Microsoft.AspNetCore.Blazor.Browser.dll" --reference "Microsoft.AspNetCore.Blazor.dll" --reference "Microsoft.Extensions.DependencyInjection.Abstractions.dll" --reference "Microsoft.Extensions.DependencyInjection.dll" --reference "Mono.Security.dll" --reference "mscorlib.dll" --reference "netstandard.dll" --reference "System.AppContext.dll" --reference "System.Collections.Concurrent.dll" --reference "System.Collections.dll" --reference "System.Collections.Immutable.dll" --reference "System.Collections.NonGeneric.dll" --reference "System.Console.dll" --reference "System.Core.dll" --reference "System.Diagnostics.Debug.dll" --reference "System.Diagnostics.DiagnosticSource.dll" --reference "System.Diagnostics.Process.dll" --reference "System.Diagnostics.StackTrace.dll" --reference "System.Diagnostics.TraceSource.dll" --reference "System.Diagnostics.Tracing.dll" --reference "System.dll" --reference "System.Globalization.dll" --reference "System.IO.Compression.dll" --reference "System.IO.dll" --reference "System.IO.FileSystem.dll" --reference "System.IO.FileSystem.Primitives.dll" --reference "System.IO.FileSystem.Watcher.dll" --reference "System.Linq.dll" --reference "System.Net.Http.dll" --reference "System.Net.NameResolution.dll" --reference "System.Net.Primitives.dll" --reference "System.Net.Requests.dll" --reference "System.Net.Sockets.dll" --reference "System.Net.WebHeaderCollection.dll" --reference "System.Reflection.dll" --reference "System.Reflection.Extensions.dll" --reference "System.Reflection.Metadata.dll" --reference "System.Reflection.Primitives.dll" --reference "System.Reflection.TypeExtensions.dll" --reference "System.Resources.ResourceManager.dll" --reference "System.Runtime.dll" --reference "System.Runtime.Extensions.dll" --reference "System.Runtime.Handles.dll" --reference "System.Runtime.InteropServices.dll" --reference "System.Runtime.InteropServices.RuntimeInformation.dll" --reference "System.Runtime.Numerics.dll" --reference "System.Runtime.Serialization.Formatters.dll" --reference "System.Runtime.Serialization.Primitives.dll" --reference "System.Security.Claims.dll" --reference "System.Security.Cryptography.Algorithms.dll" --reference "System.Security.Cryptography.OpenSsl.dll" --reference "System.Security.Cryptography.Primitives.dll" --reference "System.Security.Principal.dll" --reference "System.Text.Encoding.dll" --reference "System.Text.Encoding.Extensions.dll" --reference "System.Text.RegularExpressions.dll" --reference "System.Threading.dll" --reference "System.Threading.Tasks.dll" --reference "System.Threading.Tasks.Extensions.dll" --reference "System.Threading.Thread.dll" --reference "System.Threading.ThreadPool.dll" --reference "System.Threading.Timer.dll" --reference "System.Xml.ReaderWriter.dll" --reference "System.Xml.XmlDocument.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\log4net\2.0.8\lib\netstandard1.3\log4net.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor\0.3.0-preview1-t000\lib\netstandard2.0\Microsoft.AspNetCore.Blazor.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor.browser\0.3.0-preview1-t000\lib\netstandard2.0\Microsoft.AspNetCore.Blazor.Browser.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.extensions.dependencyinjection\2.0.0\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.extensions.dependencyinjection.abstractions\2.0.0\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.appcontext\4.1.0\lib\netstandard1.6\System.AppContext.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.collections.concurrent\4.0.12\lib\netstandard1.3\System.Collections.Concurrent.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.collections.immutable\1.2.0\lib\netstandard1.0\System.Collections.Immutable.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.collections.nongeneric\4.0.1\lib\netstandard1.3\System.Collections.NonGeneric.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.diagnostics.diagnosticsource\4.0.0\lib\netstandard1.3\System.Diagnostics.DiagnosticSource.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.diagnostics.stacktrace\4.0.1\lib\netstandard1.3\System.Diagnostics.StackTrace.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.io.filesystem.primitives\4.0.1\lib\netstandard1.3\System.IO.FileSystem.Primitives.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.linq\4.1.0\lib\netstandard1.6\System.Linq.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.net.webheadercollection\4.0.1\lib\netstandard1.3\System.Net.WebHeaderCollection.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.reflection.metadata\1.3.0\lib\netstandard1.1\System.Reflection.Metadata.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.reflection.typeextensions\4.1.0\lib\netstandard1.5\System.Reflection.TypeExtensions.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.runtime.numerics\4.0.1\lib\netstandard1.3\System.Runtime.Numerics.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.runtime.serialization.formatters\4.3.0\lib\netstandard1.4\System.Runtime.Serialization.Formatters.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.runtime.serialization.primitives\4.3.0\lib\netstandard1.3\System.Runtime.Serialization.Primitives.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.security.claims\4.0.1\lib\netstandard1.3\System.Security.Claims.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.security.cryptography.openssl\4.0.0\lib\netstandard1.6\System.Security.Cryptography.OpenSsl.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.security.cryptography.primitives\4.0.0\lib\netstandard1.3\System.Security.Cryptography.Primitives.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.security.principal\4.0.1\lib\netstandard1.0\System.Security.Principal.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.text.regularexpressions\4.1.0\lib\netstandard1.6\System.Text.RegularExpressions.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.threading\4.0.11\lib\netstandard1.3\System.Threading.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.threading.tasks.extensions\4.0.0\lib\netstandard1.0\System.Threading.Tasks.Extensions.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.threading.thread\4.0.0\lib\netstandard1.3\System.Threading.Thread.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.threading.threadpool\4.0.10\lib\netstandard1.3\System.Threading.ThreadPool.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.xml.readerwriter\4.0.11\lib\netstandard1.3\System.Xml.ReaderWriter.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.xml.xmldocument\4.0.1\lib\netstandard1.3\System.Xml.XmlDocument.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Ressources\bin\Debug\netstandard2.0\Allogaia.AppResource.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Common\bin\Debug\netstandard2.0\Allogaia.Common.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Daddoon.Blazor\Daddoon.Blazor.Extensions\bin\netstandard2.0\Daddoon.Blazor.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Ressources\bin\Debug\netstandard2.0\fr\Allogaia.AppResource.resources.dll" --linker-enabled --output "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Web.Client\obj\Debug\netstandard2.0\blazor/index.html"`

And here the output of my visual studio when failing:

```
4>Processing embedded resource linker descriptor: mscorlib.xml
4>Output action:     Link assembly: Allogaia.Web.Client, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
4>Output action:     Copy assembly: Allogaia.AppResource.resources, Version=1.0.0.0, Culture=fr, PublicKeyToken=null
4>Output action:     Copy assembly: Daddoon.Blazor, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
4>Output action:     Copy assembly: Allogaia.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
4>Output action:     Copy assembly: Allogaia.AppResource, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
4>Output action:     Copy assembly: System.Xml.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Xml.ReaderWriter, Version=4.0.11.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Threading.ThreadPool, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e
4>Output action:     Copy assembly: System.Threading.Thread, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Threading.Tasks.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
4>Output action:     Copy assembly: System.Threading, Version=4.0.11.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Text.RegularExpressions, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Security.Principal, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Security.Cryptography.Primitives, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Security.Cryptography.OpenSsl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Security.Claims, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Runtime.Serialization.Primitives, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Runtime.Serialization.Formatters, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Runtime.Numerics, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Reflection.TypeExtensions, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Reflection.Metadata, Version=1.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Net.WebHeaderCollection, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Linq, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.IO.FileSystem.Primitives, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Diagnostics.StackTrace, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Diagnostics.DiagnosticSource, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
4>Output action:     Copy assembly: System.Collections.NonGeneric, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Collections.Immutable, Version=1.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.Collections.Concurrent, Version=4.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: System.AppContext, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Copy assembly: Microsoft.Extensions.DependencyInjection.Abstractions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
4>Output action:     Copy assembly: Microsoft.Extensions.DependencyInjection, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
4>Output action:     Copy assembly: Microsoft.AspNetCore.Blazor.Browser, Version=0.3.0.0, Culture=neutral, PublicKeyToken=null
4>Output action:     Copy assembly: Microsoft.AspNetCore.Blazor, Version=0.3.0.0, Culture=neutral, PublicKeyToken=null
4>Output action:     Copy assembly: log4net, Version=2.0.8.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a
4>Output action:     Link assembly: netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
4>Output action:     Link assembly: System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e
4>Output action:     Link assembly: System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e
4>Output action:   Delete assembly: System.Data, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
4>Output action:   Delete assembly: System.Drawing, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.IO.Compression, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
4>Output action:   Delete assembly: System.IO.Compression.FileSystem, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
4>Output action:   Delete assembly: System.ComponentModel.Composition, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
4>Output action:     Link assembly: System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:   Delete assembly: System.Numerics, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
4>Output action:   Delete assembly: System.Runtime.Serialization, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e
4>Output action:   Delete assembly: System.Transactions, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
4>Output action:   Delete assembly: System.Web.Services, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:   Delete assembly: System.Xml, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e
4>Output action:   Delete assembly: System.Xml.Linq, Version=2.0.5.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
4>Output action:     Link assembly: Mono.Security, Version=2.0.5.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
4>Output action:   Delete assembly: System.ServiceModel.Internals, Version=0.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
4>Output action:     Link assembly: System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Resources.ResourceManager, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.IO, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Text.Encoding, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Collections, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Diagnostics.Debug, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Runtime.Extensions, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Globalization, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Threading.Tasks, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Runtime.InteropServices, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Text.Encoding.Extensions, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.IO.FileSystem, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Security.Cryptography.Algorithms, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Runtime.Handles, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Reflection, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Reflection.Primitives, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Reflection.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Diagnostics.Tracing, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Net.Sockets, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Net.Requests, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Net.Primitives, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Net.NameResolution, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Threading.Timer, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.IO.FileSystem.Watcher, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Console, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Diagnostics.TraceSource, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Diagnostics.Process, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>Output action:     Link assembly: System.Runtime.InteropServices.RuntimeInformation, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
4>
4>Unhandled Exception: Microsoft.Extensions.CommandLineUtils.CommandParsingException: Unrecognized option '--embedded-resources-souce'
4>   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.HandleUnexpectedArg(CommandLineApplication command, String[] args, Int32 index, String argTypeName)
4>   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
4>   at Microsoft.AspNetCore.Blazor.Build.Program.Main(String[] args)
4>C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor.build\0.3.0-preview1-t000\targets\Blazor.MonoRuntime.targets(602,5): error MSB3073: La commande "dotnet "C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor.build\0.3.0-preview1-t000\targets\../tools/Microsoft.AspNetCore.Blazor.Build.dll" build obj\Debug\netstandard2.0\Allogaia.Web.Client.dll --html-page "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Web.Client\wwwroot/Index.html" --reference "Allogaia.AppResource.dll" --reference "Allogaia.AppResource.resources.dll" --reference "Allogaia.Common.dll" --reference "Daddoon.Blazor.dll" --reference "log4net.dll" --reference "Microsoft.AspNetCore.Blazor.Browser.dll" --reference "Microsoft.AspNetCore.Blazor.dll" --reference "Microsoft.Extensions.DependencyInjection.Abstractions.dll" --reference "Microsoft.Extensions.DependencyInjection.dll" --reference "Mono.Security.dll" --reference "mscorlib.dll" --reference "netstandard.dll" --reference "System.AppContext.dll" --reference "System.Collections.Concurrent.dll" --reference "System.Collections.dll" --reference "System.Collections.Immutable.dll" --reference "System.Collections.NonGeneric.dll" --reference "System.Console.dll" --reference "System.Core.dll" --reference "System.Diagnostics.Debug.dll" --reference "System.Diagnostics.DiagnosticSource.dll" --reference "System.Diagnostics.Process.dll" --reference "System.Diagnostics.StackTrace.dll" --reference "System.Diagnostics.TraceSource.dll" --reference "System.Diagnostics.Tracing.dll" --reference "System.dll" --reference "System.Globalization.dll" --reference "System.IO.Compression.dll" --reference "System.IO.dll" --reference "System.IO.FileSystem.dll" --reference "System.IO.FileSystem.Primitives.dll" --reference "System.IO.FileSystem.Watcher.dll" --reference "System.Linq.dll" --reference "System.Net.Http.dll" --reference "System.Net.NameResolution.dll" --reference "System.Net.Primitives.dll" --reference "System.Net.Requests.dll" --reference "System.Net.Sockets.dll" --reference "System.Net.WebHeaderCollection.dll" --reference "System.Reflection.dll" --reference "System.Reflection.Extensions.dll" --reference "System.Reflection.Metadata.dll" --reference "System.Reflection.Primitives.dll" --reference "System.Reflection.TypeExtensions.dll" --reference "System.Resources.ResourceManager.dll" --reference "System.Runtime.dll" --reference "System.Runtime.Extensions.dll" --reference "System.Runtime.Handles.dll" --reference "System.Runtime.InteropServices.dll" --reference "System.Runtime.InteropServices.RuntimeInformation.dll" --reference "System.Runtime.Numerics.dll" --reference "System.Runtime.Serialization.Formatters.dll" --reference "System.Runtime.Serialization.Primitives.dll" --reference "System.Security.Claims.dll" --reference "System.Security.Cryptography.Algorithms.dll" --reference "System.Security.Cryptography.OpenSsl.dll" --reference "System.Security.Cryptography.Primitives.dll" --reference "System.Security.Principal.dll" --reference "System.Text.Encoding.dll" --reference "System.Text.Encoding.Extensions.dll" --reference "System.Text.RegularExpressions.dll" --reference "System.Threading.dll" --reference "System.Threading.Tasks.dll" --reference "System.Threading.Tasks.Extensions.dll" --reference "System.Threading.Thread.dll" --reference "System.Threading.ThreadPool.dll" --reference "System.Threading.Timer.dll" --reference "System.Xml.ReaderWriter.dll" --reference "System.Xml.XmlDocument.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\log4net\2.0.8\lib\netstandard1.3\log4net.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor\0.3.0-preview1-t000\lib\netstandard2.0\Microsoft.AspNetCore.Blazor.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.aspnetcore.blazor.browser\0.3.0-preview1-t000\lib\netstandard2.0\Microsoft.AspNetCore.Blazor.Browser.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.extensions.dependencyinjection\2.0.0\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\microsoft.extensions.dependencyinjection.abstractions\2.0.0\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.appcontext\4.1.0\lib\netstandard1.6\System.AppContext.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.collections.concurrent\4.0.12\lib\netstandard1.3\System.Collections.Concurrent.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.collections.immutable\1.2.0\lib\netstandard1.0\System.Collections.Immutable.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.collections.nongeneric\4.0.1\lib\netstandard1.3\System.Collections.NonGeneric.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.diagnostics.diagnosticsource\4.0.0\lib\netstandard1.3\System.Diagnostics.DiagnosticSource.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.diagnostics.stacktrace\4.0.1\lib\netstandard1.3\System.Diagnostics.StackTrace.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.io.filesystem.primitives\4.0.1\lib\netstandard1.3\System.IO.FileSystem.Primitives.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.linq\4.1.0\lib\netstandard1.6\System.Linq.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.net.webheadercollection\4.0.1\lib\netstandard1.3\System.Net.WebHeaderCollection.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.reflection.metadata\1.3.0\lib\netstandard1.1\System.Reflection.Metadata.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.reflection.typeextensions\4.1.0\lib\netstandard1.5\System.Reflection.TypeExtensions.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.runtime.numerics\4.0.1\lib\netstandard1.3\System.Runtime.Numerics.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.runtime.serialization.formatters\4.3.0\lib\netstandard1.4\System.Runtime.Serialization.Formatters.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.runtime.serialization.primitives\4.3.0\lib\netstandard1.3\System.Runtime.Serialization.Primitives.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.security.claims\4.0.1\lib\netstandard1.3\System.Security.Claims.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.security.cryptography.openssl\4.0.0\lib\netstandard1.6\System.Security.Cryptography.OpenSsl.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.security.cryptography.primitives\4.0.0\lib\netstandard1.3\System.Security.Cryptography.Primitives.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.security.principal\4.0.1\lib\netstandard1.0\System.Security.Principal.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.text.regularexpressions\4.1.0\lib\netstandard1.6\System.Text.RegularExpressions.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.threading\4.0.11\lib\netstandard1.3\System.Threading.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.threading.tasks.extensions\4.0.0\lib\netstandard1.0\System.Threading.Tasks.Extensions.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.threading.thread\4.0.0\lib\netstandard1.3\System.Threading.Thread.dll" --embedded-resources-source "C:\Users\blyzz\.nuget\packages\system.threading.threadpool\4.0.10\lib\netstandard1.3\System.Threading.ThreadPool.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.xml.readerwriter\4.0.11\lib\netstandard1.3\System.Xml.ReaderWriter.dll" --embedded-resources-source "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\system.xml.xmldocument\4.0.1\lib\netstandard1.3\System.Xml.XmlDocument.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Ressources\bin\Debug\netstandard2.0\Allogaia.AppResource.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Common\bin\Debug\netstandard2.0\Allogaia.Common.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Daddoon.Blazor\Daddoon.Blazor.Extensions\bin\netstandard2.0\Daddoon.Blazor.dll" --embedded-resources-source "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Ressources\bin\Debug\netstandard2.0\fr\Allogaia.AppResource.resources.dll" --linker-enabled --output "C:\Appizeo\Dev\Allogaia\Allogaia\Allogaia.Web.Client\obj\Debug\netstandard2.0\blazor/index.html"" s'est arrêtée avec le code -532462766.
4>Génération du projet "Allogaia.Web.Client.csproj" terminée -- ÉCHEC.
========== Régénération globale : 3 a réussi, 1 a échoué, 0 a été ignoré ==========
```

I think this issue will shortly come over on upcoming projects.

I would suggest to read input files from another kind of source if possible (a file ?) or maybe execute the command from powershell at build, but i'm not even sure there is not the same type of limitation present.

--- a/src/Microsoft.AspNetCore.Blazor.Build/Cli/Commands/BuildIndexHtmlCommand.cs
+++ b/src/Microsoft.AspNetCore.Blazor.Build/Cli/Commands/BuildIndexHtmlCommand.cs
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.IO;
 using Microsoft.Extensions.CommandLineUtils;
 
 namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
@@ -18,9 +19,9 @@ namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
                 "The path from the _bin folder to a given referenced dll file (typically just the dll name)",
                 CommandOptionType.MultipleValue);
 
-            var embeddedResourcesSources = command.Option("--embedded-resources-source",
-                "The path to an assembly that may contain embedded resources (typically a referenced assembly in its pre-linked state)",
-                CommandOptionType.MultipleValue);
+            var embeddedResourcesFile = command.Option("--embedded-resources",
+                "The path to a file that lists the paths of .NET assemblies that may contain embedded resources (typically, referenced assemblies in their pre-linked states)",
+                CommandOptionType.SingleValue);
 
             var outputPath = command.Option("--output",
                 "Path to the output file",
@@ -44,11 +45,15 @@ namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
 
                 try
                 {
+                    var embeddedResourcesSources = embeddedResourcesFile.HasValue()
+                        ? File.ReadAllLines(embeddedResourcesFile.Value())
+                        : Array.Empty<string>();
+
                     IndexHtmlWriter.UpdateIndex(
                         clientPage.Value(),
                         mainAssemblyPath.Value,
                         references.Values.ToArray(),
-                        embeddedResourcesSources.Values.ToArray(),
+                        embeddedResourcesSources,
                         linkerEnabledFlag.HasValue(),
                         outputPath.Value());
                     return 0;

--- a/src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets
+++ b/src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets
@@ -236,6 +236,9 @@
       <!-- /obj/<<configuration>>/<<targetframework>>/blazor/inputs.index.cache -->
       <BlazorBuildIndexInputsCache>$(BlazorIntermediateOutputPath)inputs.index.cache</BlazorBuildIndexInputsCache>
 
+      <!-- /obj/<<configuration>>/<<targetframework>>/blazor/embedded.resources.txt -->
+      <BlazorEmbeddedResourcesConfigFilePath>$(BlazorIntermediateOutputPath)embedded.resources.txt</BlazorEmbeddedResourcesConfigFilePath>
+
     </PropertyGroup>
 
     <PropertyGroup Label="Final output paths">
@@ -597,9 +600,16 @@
     </ItemGroup>
     <PropertyGroup>
       <_LinkerEnabledFlag Condition="'$(_BlazorShouldLinkApplicationAssemblies)' != ''">--linker-enabled</_LinkerEnabledFlag>
+      <_EmbeddedResourcesArg Condition="'@(_UnlinkedAppReferencesPaths)' != ''">--embedded-resources &quot;$(BlazorEmbeddedResourcesConfigFilePath)&quot;</_EmbeddedResourcesArg>
     </PropertyGroup>
 
-    <Exec Command="$(BlazorBuildExe) build @(IntermediateAssembly) --html-page &quot;$(BlazorIndexHtml)&quot; @(_AppReferences->'--reference &quot;%(Identity)&quot;', ' ') @(_UnlinkedAppReferencesPaths->'--embedded-resources-source &quot;%(Identity)&quot;', ' ') $(_LinkerEnabledFlag) --output &quot;$(BlazorIndexHtmlOutputPath)&quot;" />
+    <WriteLinesToFile
+      Condition="'@(_UnlinkedAppReferencesPaths)' != ''"
+      File="$(BlazorEmbeddedResourcesConfigFilePath)"
+      Lines="@(_UnlinkedAppReferencesPaths)"
+      Overwrite="true" />
+
+    <Exec Command="$(BlazorBuildExe) build @(IntermediateAssembly) --html-page &quot;$(BlazorIndexHtml)&quot; @(_AppReferences->'--reference &quot;%(Identity)&quot;', ' ') $(_EmbeddedResourcesArg) $(_LinkerEnabledFlag) --output &quot;$(BlazorIndexHtmlOutputPath)&quot;" />
 
     <ItemGroup Condition="Exists('$(BlazorIndexHtmlOutputPath)')">
       <_BlazorIndex Include="$(BlazorIndexHtmlOutputPath)" />

